mlblocks_primitives/keras.Sequential.LSTMTimeSeriesRegressor.json

```diff
@@ -0,0 +1,114 @@
+{
+    "name": "keras.Layers.LSTM",
+    "author": "Ihssan Tinawi <itinawi@mit.edu>",
+    "documentation": "https://keras.io/layers/recurrent/#lstm",
+    "description": "Long Short Term Memory Layer",
+    "classifiers": {
+        "type": "estimator",
+        "subtype": "regressor"
+    },
+    "modalities": [],
+    "primitive": "mlprimitives.adapters.keras.Sequential",
+    "fit": {
+        "method": "fit",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            },
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "produce": {
+        "method": "predict",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "hyperparameters": {
+        "fixed": {
+            "dense_units": {
+                "type": "int",
+                "description": "Number of classes"
+            },
+            "dense_activation": {
+                "type": "str",
+                "default": "tanh"
+            },
+            "optimizer": {
+                "type": "str",
+                "default": "keras.optimizers.Adam"
+            },
+            "loss": {
+                "type": "str",
+                "default": "keras.losses.mean_squared_error"
+            },
+            "metrics": {
+                "type": "list",
+                "default": [
+                    "accuracy"
+                ]
+            },
+            "layers": {
+                "type": "list",
+                "default": [
+                    {
+                        "class": "keras.layers.Input",
+                        "parameters": {
+                            "shape": "input_shape"
+                        }
+                    },
+                    {
+                        "class": "keras.layers.Dropout",
+                        "parameters": {
+                            "rate": "dropout_rate"
+                        }
+                    },
+                    {
+                        "class": "keras.layers.LSTM",
+                        "parameters": {
+                            "units": "lstm_input_dim"
```

Comment:
Please call this `lstm_units`, just like we use `dense_units` or `dropout_rate`, indicating both the name of the layer and the name of parameter.